tests/prescribed_temperature_in_field.prm

```diff
@@ -0,0 +1,64 @@
+## First we set up a unit length box model with uniform everything.
+set Dimension   = 2
+set End time    = 10000
+# Load the signal library.
+set Additional shared libraries = /Users/danieldouglas/software/aspect/cookbooks/prescribed_temperature/./libprescribed_temperature.so
```

Comment:
We typically have the line for shared libraries at the top of the prm file. I think this might be just convention, but I find it useful to see the included libraries listed at the top.